### PR TITLE
Recruitifyv1

### PR DESCRIPTION
**Pull request by Tarik-Anowar**

### CODE REVIEW

**Comment by priyam-03** [comment]
1. In the get job list there will be job role , company name , location 
2. Clicking that it opens the job page where all the details will be there 
3. if applicants , he/she can see applied or not , (if not can apply)
4. if recruiter can show details of all applicants
5. update profile

File: backend/uploads/2024-06-14T06-31-36.145Z-IMG_0432_2.JPG
Comment (priyam-03) [inline backend/uploads/2024-06-14T06-31-36.145Z-IMG_0432_2.JPG:1]: Please remove this picture. Or else I will not merge it.

Comment (Tarik-Anowar) [reply backend/uploads/2024-06-14T06-31-36.145Z-IMG_0432_2.JPG:1]: I will do it in the next commit

**Comment by Tarik-Anowar** [comment]
Added required fields. Some more additional features like edit/delete jobs profile and need to some real time communications...

**Review by priyam-03 — COMMENTED** [review]
update the resume part

File: backend/.gitignore
Comment (priyam-03) [inline backend/.gitignore:1]: Nice code

**Review by priyam-03 — COMMENTED** [review]
Great Contribution

**Review by priyam-03 — COMMENTED** [review]
Scrolling issue in chatting need to be addressed.

**Pull request merged**